cmd/testworkflow-toolkit/artifacts/junit_post_processor.go

```diff
@@ -57,19 +57,20 @@ func (p *JUnitPostProcessor) Add(path string) error {
 	if ok := isXMLFile(stat); !ok {
 		return nil
 	}
-	// Read only the first 8KB of data
-	const BYTE_SIZE_8KB = 8 * 1024
-	xmlData := make([]byte, BYTE_SIZE_8KB)
-	n, err := file.Read(xmlData)
-	if err != nil && err != io.EOF {
+
+	xmlData, err := io.ReadAll(file)
```

Comment:
You can i.e. use `io.Read` to 8KB buffer, and later - if it's filled fully - ReadAll and concatenate with that buffer